crates/db/src/commands.rs

```diff
@@ -1,5 +1,5 @@
 use ethers::types::{Address, U256};
-use iron_types::{events::Tx, TokenBalance};
+use iron_types::{events::Tx, TokenBalance, NftToken};
```

Comment:
same here. `Erc721`. I probably need to rename some stuff regarding the erc20 ones as well, but that's on me